packages/salesforcedx-apex-debugger/package.json

```diff
@@ -12,6 +12,8 @@
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "40.7.0",
     "faye": "1.1.2",
+    "request": "2.81.0",
```

Comment:
Let's not use the request package. I've heard bad things about it (being heavyweight).
Use request-light as recommended at https://code.visualstudio.com/docs/extensionAPI/extension-manifest#_useful-node-modules